hw2/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Request, Response } from 'express';
+import { addUser, deleteUserById, getAutoSuggestUsers, getUserById, getUserList, updateUserById } from './utils';
+import { validateUser } from './validators/user.validator';
+import { userSchema } from './models/user.schema';
+import express = require('express');
+
+const app = express();
+const router = express.Router();
+
+app.use(express.json());
+app.use('/api', router);
+
+router.get('/getUserList', (req: Request, res: Response) => {
```

Comment:
It's more useful to create a separate file for routing, to split `app configuration` and `endpoints routing`.

Reply:
Also, move the endpoint handlers implementation to a separate file (aka controller).